src/main/java/org/apereo/openlrs/controllers/caliper/CaliperController.java

```diff
@@ -143,6 +234,21 @@ public JsonNode getByIdHandler (
 
     @RequestMapping(value = "about", method = RequestMethod.GET, produces = "application/json;charset=utf-8")
     public About about() {
-        return new About(version);
+        About aboutCaliper = new About(version);
+        aboutCaliper.setExtensions(new HashMap<URI, String>(){
+            {
+                String hostAddress;
+
+                try{
+                    hostAddress = InetAddress.getLocalHost().getHostAddress();
+                } catch (Exception e) {
+                    hostAddress = "unknown";
```

Comment:
In general, it's always good to log errors within a catch clause, but in this case, I don't think it's necessary.